### PR TITLE
added personal attacks to unacceptable

### PR DESCRIPTION
**Pull request by TzviyaSiegman**
* added personal attacks to list of unacceptable behaviors

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/PWETF/pull/184.html" title="Last updated on Jul 13, 2021, 7:25 PM UTC (0e4e31f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/PWETF/184/46619a6...0e4e31f.html" title="Last updated on Jul 13, 2021, 7:25 PM UTC (0e4e31f)">Diff</a>

### CODE REVIEW

**Comment by brewerj** [comment]
Adding "personal attacks" to the list of unaccepted behaviors addresses part of the problem. However, sometimes comments can be against an organization, for instance by attributing malicious intent to representatives of an organization. 

Not sure how to best capture that. One possibility would be: 

"Attacks against individuals or organizations" 

- Judy

File: index.html [hunk 1]
```diff
@@ -265,6 +265,8 @@ <h3 id="unacceptablebehavior">
           </li>
           <li>Deliberate intimidation.
           </li>
+	 <li>Personal attacks.
```
Comment (hober) [inline index.html:268]: whitespace is not the same as the other lines?

File: index.html [hunk 2]
```diff
@@ -276,7 +278,7 @@ <h3 id="unacceptablebehavior">
           is not limited to, various common methods of engaging in bad
           faith discourse such as:<ul>
               <!-- https://geekfeminism.wikia.org/wiki/Concern_troll -->
-              <li>disingenuously expressesing concern in order to
+              <li>disingenuously expressing concern in order to
```
Comment (hober) [inline index.html:281]: oops, thanks for catching this!

**Review by hober — APPROVED** [review]
Looks good to me!

**Comment by TzviyaSiegman** [comment]
Anon commentator suggested adapting to "verbal attacks or denigration of individuals or organizations"

**Comment by jspellman** [comment]
"Verbal or written attacks, insinuations, or denigration of individuals or organizations."

Insinuations are hard to prove, but it is helpful to have that language to call out unacceptable behavior.

**Comment by LJWatson** [comment]
Discussed in the [PWE meeting today](https://www.w3.org/2021/07/27-pwe-minutes.html).

It was agreed that, since this PR has conflicts that need fixing before it can be merged, it would make sense to refine the language at the same time so that attacks on people and organisations were both called out.

**Pull request merged**